tests/test_ec2_manager.py

```diff
@@ -11,11 +11,72 @@ def em():
 
 
 def test_list_security_groups(em):
-    for g_id in em.list_security_groups():
-        print(g_id['GroupName'])
+    pprint(em.list_security_groups())
     assert em.list_security_groups()
 
 
+def test_list_instances(em):
+    pprint(em.list_instances())
+    assert type(em.list_instances()) == set
+
+
+def test_list_subnet(em):
+    pprint(em.list_subnet())
+
+
+def test_delete_redundant_security_groups(em):
+    em.delete_redundant_security_groups()
+
+
+def test_list_redundant_security_group_ids(em):
+    pprint(em.list_redundant_security_group_ids())
+    assert type(em.list_redundant_security_group_ids()) == set
+
+
+def test_list_vpc_sg_eni_subnets(em):
+    result = em.list_vpc_sg_eni_subnets()
+    pprint(result)
+    assert type(result) == list
+    assert len(result) == len(em.list_security_groups())
+    if len(result) != 0:
+        assert type(result[0]) == dict
+
+
+def test_get_sg_ids_with_vpc_id(em):
+    vpc_id = 'vpc-04863aaf1ea9a3b35'
```

Comment:
I think it's better to make getting default vpc id method and utilizing it to avoid defining specific vpc id here.

this id is unnecessary to users who use Baram.

Reply:
Thanks for giving sustainable opinion.